environments/pna/public.yml

```diff
@@ -103,3 +103,5 @@ localsettings:
   ENABLE_PRELOGIN_SITE: False
   ENTERPRISE_MODE: True
   RESTRICT_DOMAIN_CREATION: True
+
+letsencrypt_cchq_ssl: True
```

Comment:
should put this in `proxy.yml` - will also need to update [ProxyConfig](https://github.com/dimagi/commcare-cloud/blob/1e0d1ecbdb4f2377fb4ac34d0e6fd8698f879440/src/commcare_cloud/environment/schemas/proxy.py#L4)

Reply:
Could also validate that the nginx cert vars are empty if this is true